### PR TITLE
fix(ViewProps): Update layout-only view props

### PR DESCRIPTION
**Pull request by rozele**
With these view props missing from the layout-only list, extra views are being created that are truly layout-only.

### CODE REVIEW

File: ReactWindows/ReactNative.Shared/UIManager/ViewProps.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared/UIManager/ViewProps.cs]: sort.

File: ReactWindows/ReactNative.Shared/UIManager/ViewProps.cs [hunk 2]
```diff
@@ -180,13 +189,24 @@ public static class ViewProps
         /// <summary>
         /// Checks if the property key is layout-only.
         /// </summary>
-        /// <param name="key">The key.</param>
+        /// <param name="props">The prop collection.</param>
+        /// <param name="prop">The React prop.</param>
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared/UIManager/ViewProps.cs]: "The prop name"

File: ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyOptimizer.cs [hunk 3]
```diff
@@ -1,8 +1,9 @@
 ﻿#define ENABLED
 
-using System;
 using System.Collections.Generic;
+#if ENABLED
```
Comment (matthargett) [inline ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyOptimizer.cs]: maybe a more unique symbol name?

Comment (rozele) [inline ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyOptimizer.cs]: Certainly.

**Comment by rozele** [comment]
This PR should be cherry-picked back to version 0.40-0.45.

**Pull request merged**